specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5643 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure IoT Central",
+    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale.",
+    "version": "1.2-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{subdomain}.{baseDomain}",
+    "parameters": [
+      {
+        "$ref": "#/parameters/BaseDomain"
+      },
+      {
+        "$ref": "#/parameters/Subdomain"
+      }
+    ]
+  },
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "definitions": {
+    "ADGroupUser": {
+      "x-ms-discriminator-value": "adGroup",
+      "allOf": [
+        {
+          "$ref": "#/definitions/User"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "description": "The AAD tenant ID of the AD Group.",
+          "type": "string"
+        },
+        "objectId": {
+          "description": "The AAD object ID of the AD Group.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "tenantId",
+        "objectId"
+      ]
+    },
+    "AnyValue": {
+      "description": "Can be anything: string, number, array, object, etc. (except `null`)"
+    },
+    "ApiToken": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Permission"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID of the API token.",
+          "type": "string",
+          "readOnly": true
+        },
+        "token": {
+          "description": "Value of the API token.",
+          "type": "string",
+          "readOnly": true
+        },
+        "expiry": {
+          "description": "String-formatted date representing the time when the token expires.",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "ApiTokenCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of API tokens.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiToken"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of API tokens.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
```

Comment:
Schema should have a description or title.

Reply:
Fixed.